Procfile

```diff
@@ -1,3 +1,3 @@
 release: python manage.py migrate
 web: gunicorn hc.wsgi
-worker: ./manage.py sendalerts
+worker: ./manage.py ensuretriggers && ./manage.py sendalerts
```

Comment:
@sir3n-sn shouldn't we also add ./manage.py sendreports to this? So we can be able to send email reports on checks
Cc: @VincentHokie